Rendera/include/scene.hpp

```diff
@@ -4,7 +4,7 @@
 #include "sphere.hpp"
 #include "intersection.hpp"
 #include "camera.hpp"
-
+#include "cube.hpp"
 class Hitinfo{
```

Comment:
This is a One Definition Rule (ODR) violation. You can only have definition for a class. Why did you define `HitInfo` multiple times?